even_fib_nums.js

```diff
@@ -6,21 +6,46 @@
  * @return {Number} sum
  */
 function _sumFibs( maxFibValue ) {
+  let a = 0;
+  let b = 1;
+  let c = 0;
   let sum = 0;
 
   // do your work here
-
+for (let i = 1; i <= maxFibValue; i++) {
```

Comment:
Instead of the second clause using `i <= maxFibValue` use `c <= maxFibValue` or event better try doing this with a do-while loop to cut down on unnecessary code.